### DISCUSSION TITLE
The status of whole project

### DISCUSSION THREAD

**Discussion by aefn** [loopbackio/loopback-next#9631] [Q&A]
Dear All,

I have recently become familiar with LoopBack and I would like to express my gratitude to everyone who has contributed to this excellent framework. However, I have noticed something that seems a bit strange to me. It appears that the project is not being actively maintained, or at the very least, it gives off that impression.

Does anyone know what the plan is for the future of LoopBack? The StrongLoop blog hasn't been updated since 2020, and the main contributors to the project haven't collaborated in quite some time.

Once again, I would like to sincerely appreciate all your efforts and kindly request that you continue working on this project. I'm hopeful that I will receive some positive news.

**Comment by shubhamp-sf** [comment]
@raymondfeng @dhmlau @achrinza ^

**Comment by raymondfeng** [comment]
LoopBack is mature framework and it is being actively governed by OpenJS foundation and maintained by volunteer contributors instead of paid employees. Some of us use LoopBack in our own projects. I don't think there are major development of new features for LoopBack at the moment. We fix bugs, update dependencies, and address security concerns. We also see contributions of modules from communities too.

**Reply by HarshalNathe** [reply]
@raymondfeng can you please provide suggestion, that going with LoopBack4 framework is not an issue at this time? Just wanted to get some understanding on the future release of the loopback. As the EOL of LoopBack4 is Apr 2026 (minimum), so not expecting major changes in upcoming releases of loopback, like I seen the transition of loopback from LoopBack3 to LoopBack4. (it was a major change)
Thanks!
